core.d.ts

```diff
@@ -114,11 +114,14 @@ console.log(await fileTypeFromBlob(blob));
 export declare function fileTypeFromBlob(blob: Blob): Promise<FileTypeResult | undefined>;
 
 /**
-Function that allows specifying custom detection mechanisms.
+A custom detector is a function that allows specifying custom detection mechanisms.
```

Comment:
This sentence does not actually say much.

```suggestion
A custom file type detector.
```

Reply:
Simplify file type detector description